components/core/src/clp_s/AggregationSink.cpp

```
@@ -0,0 +1,63 @@
+                value
+        );
+    }
+    m_results.push_back(document.extract());
+}
+
+auto ResultsCacheSink::finish() -> ErrorCode {
+    if (m_results.empty()) {
+        return ErrorCode::ErrorCodeSuccess;
+    }
+
+    try {
```

Cppcheck (2.21.0)

[style] The function 'get_bucket_size_ms' is never used.

(unusedFunction)